app/services/foreman_salt/fact_parser.rb

```diff
@@ -0,0 +1,78 @@
+module ForemanSalt
+  class FactParser < ::PuppetFactParser
```

Comment:
so maybe we could extract common bits to some generic class and PuppetFactParser should inherit from it?